src/libudev/libudev.pc.in

```diff
@@ -16,4 +16,5 @@ Name: libudev
 Description: Library to access udev device information
 Version: {{PROJECT_VERSION}}
 Libs: -L${libdir} -ludev
+Libs.private: -lrt
```

Comment:
should we have -pthread here too?

Reply:
It looks like cryptsetup's build system is already passing -pthread, but it's possible other packages do not do this. I'll go ahead and add it.